modules/geo-layers/src/tile-3d-layer/tile-3d-layer.js

```diff
@@ -67,84 +64,68 @@ export default class Tile3DLayer extends CompositeLayer {
     return info;
   }
 
-  async _loadTileset(tilesetUrl, fetchOptions, ionMetadata) {
-    const response = await fetch(tilesetUrl, fetchOptions);
-    const tilesetJson = await response.json();
-
-    const loadOptions = this.getLoadOptions();
-
-    const tileset3d = new Tileset3D(tilesetJson, tilesetUrl, {
-      onTileLoad: tileHeader => {
-        this.props.onTileLoad(tileHeader);
-        this._updateTileset(tileset3d);
-      },
+  async _loadTileset(tilesetUrl) {
+    const {loader, loadOptions} = this.props;
+    const tilesetJson = await load(tilesetUrl, loader, loadOptions);
```

Comment:
What if loader is not supplied? Add helpful warning to help users cope with the breaking change.